latch_cli/docker_utils/__init__.py

```diff
@@ -323,15 +325,6 @@ def copy_file_commands(wf_type: WorkflowType) -> List[DockerCmdBlock]:
             "",
             "copy .latch/snakemake_jit_entrypoint.py /root/snakemake_jit_entrypoint.py",
         ]
-    elif wf_type == WorkflowType.nextflow:
```

Comment:
since we are deleting this stuff, can you update the wiki as well to reflect the changes. 

i believe a Dockerfile gets created if one doesn't exist? if that is the case, we can prob just remove the Dockerfile generation from the Nextflow docs all together.